docs/V3/concepts/oracle.md

```diff
@@ -7,80 +7,94 @@ title: Oracle
 Unfamiliar with oracles? check out the Ethereum Foundation's [oracle overview](https://ethereum.org/en/developers/docs/oracles/) first.
 :::
 
-All Uniswap v3 pools can serve as _oracles_, offering access to historical price and liquidity data. This capability unlocks a wide range of on-chain use cases.
+# Overview
+Oracles are smart contracts that communicate price feed information between assets traded on-chain. Their goal is to signal the trading price between a pair of assets and are often used as inputs to another contract’s execution. DeFi projects that move billions in value rely on price oracles to feed untampered data to their protocols.
 
-Historical data is stored as an array of observations. At first, each pool tracks only a single observation, overwriting it as blocks elapse. This limits how far into the past users may access data. However, any party willing to pay the transaction fees may increase the number of tracked observations (up to a maximum of `65535`), expanding the period of data availability to 9 days or more.
+Uniswap is an excellent data source for price oracles because of the sheer volume of trades. Exchanges with large volumes are likely to follow the “true” spot price of a trading pair and are more resistant to bad actors. (“True” refers to the hypothetical fair market value across all markets.)
```

Comment:
This is a fairly technical document. Rhetoric involving the volume of trades and accuracy isn't really helpful, Similarly, ideas of "true" price will lead to confusing without explicitly discussing margin of error in terms of tx cost and swap fees. I would not even introduce the concept of true price.